certbot/client.py

```diff
@@ -240,6 +241,9 @@ def __init__(self, config, account_, auth, installer, acme=None):
                 auth, self.acme, self.account, self.config.pref_challs)
         else:
             self.auth_handler = None
+        if sys.version_info[:2] == (3, 3):
```

Comment:
I don't think we should do this in this file and I instead think we should do it in code that Certbot (almost) always runs such as code in `main.py`. The reason for this is some user's might not get this warning. For instance, someone running `certbot renew` only gets this message when their certificate is attempted to be renewed which only happens every 60 days by default. Since we have been doing a release every month, some user's in this common case may miss the warning entirely.